source/test/F0.Minesweeper.Components.Tests/F0.Minesweeper.Components.Tests.csproj

```diff
@@ -5,12 +5,18 @@
 	</PropertyGroup>
 
 	<ItemGroup>
-		<PackageReference Include="bunit.web" Version="1.0.0-beta-10" />
-		<PackageReference Include="bunit.xunit" Version="1.0.0-beta-10" />
+		<PackageReference Include="bunit.web" Version="1.0.0-beta-11" />
+		<PackageReference Include="bunit.xunit" Version="1.0.0-beta-11" />
+		<PackageReference Include="FluentAssertions" Version="5.10.3" />
```

Comment:
It seems that we need both _FluentAssertions_ and _Moq_ in both test projects (_Logic-Tests_ & _Components-Tests_).

I suggest moving both `PackageReference`s from both `*.Tests.csproj` to the parent's `/test/Directory.Build.props`, so that we use the same version in all test projects.
Unfortunately, tooling doesn't consider the `Directory.Build.props` file yet.